app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -206,10 +206,20 @@ def qualification_to_hash(qualification)
         award_year: qualification.award_year,
         institution_details: institution_details(qualification),
         awarding_body: qualification.awarding_body,
-        equivalency_details: qualification.equivalency_details,
+        equivalency_details: composite_equivalency_details(qualification),
       }
     end
 
+    def composite_equivalency_details(qualification)
+      details = [
+        ("Naric: #{qualification.naric_reference}" if qualification.naric_reference),
+        ("- #{qualification.comparable_uk_qualification} -" if qualification.comparable_uk_qualification),
```

Comment:
Is it right to bake the dashes into this string? What if it's missing. Should we rather `join(' - ')`?

Reply:
If it's missing, the `if qualification.comparable_uk_qualification` makes it `nil`. But we might as well use `join(' - ')`. :+1: